assets/rich_text_components_definitions.ts

```diff
@@ -109,6 +112,27 @@ export = {
       "default_value": ""
     }]
   },
+  "Concept": {
+    "backend_id": "concept",
+    "category": "Basic Input",
+    "description": "A link to the concept card of the linked skill.",
+    "frontend_id": "concept",
+    "tooltip": "Insert Concept Card Link",
+    "icon_data_url": "/rich_text_components/Concept/Concept.png",
+    "is_complex": false,
+    "requires_fs": false,
+    "is_lesson_related": true,
+    "is_block_element": false,
+    "customization_arg_specs": [{
+      "name": "skill_summary",
+      "description": "Click on skill to view concept card.",
+      "schema": {
+        "type": "custom",
+        "obj_type": "SkillSelector"
+      },
+      "default_value": ""
+    }]
+  },
```

Comment:
Please move this after the Collapsible to keep the alphabetical ordering.

Reply:
Done